incubator/kafka/README.md

```diff
@@ -0,0 +1,113 @@
+cluster in its `requirement.yaml`. The chart can be customized using the
+following configurable parameters:
+
+| Parameter               | Description                        | Default                                                    |
+| ----------------------- | ---------------------------------- | ---------------------------------------------------------- |
+| `Name`                  | Kafka master name                  | `kf`                                                       |
+| `Image`                 | Kafka Container image name         | `solsson/kafka`                                            |
+| `ImageTag`              | Kafka Container image tag          | `0.10.0.1`                                                 |
+| `ImagePullPolicy`       | Kafka Container pull policy        | `Always`                                                   |
+| `Replicas`              | Kafka Brokers                      | `3`                                                        |
+| `Component`             | Kafka k8s selector key             | `kafka`                                                    |
+| `Cpu`                   | Kafka container requested cpu      | `100m`                                                     |
+| `Memory`                | Kafka container requested memory   | `512Mi`                                                    |
+| `MaxCpu`                | Kafka container cpu limit          | `200m`                                                     |
+| `MaxMemory`             | Kafka container memory limit       | `1024Mi`                                                   |
+| `DataDirectory`         | Kafka data directory               | `/opt/kafka/data`                                          |
+| `Storage`               | Kafka Persistent volume size       | `1Gi`                                                      |
+| `zookeeper.Name`        | Name of the Zookeeper Service      | `zk`                                                       |
+
+Specify parameters using `--set key=value[,key=value]` argument to `helm install`
+
+Alternatively a YAML file that specifies the values for the parameters can be provided like this:
+
+```bash
+$ helm install --name my-release -f values.yaml incubator/kafka
+```
+
+### Connecting to Kafka
+
+You can connect to Kafka by running a simple pod in the K8s cluster like this with a configuration like this:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: testclient
+  namespace: kafka
+spec:
+  containers:
+  - name: kafka
+    image: solsson/kafka:0.10.0.1
+    command:
+      - sh
+      - -c
+      - "exec tail -f /dev/null"
+```
+
+Once you have the testclient pod above running, you can list all kafka
+topics with:
+
+` kubectl exec testclient -- ./bin/kafka-topics.sh --zookeeper
+my-release-zk:2181 --list`
+
+Where `my-release` is the name of your helm release.
+
+## Known Limitations
+
+* Namespace creation is not automated
```

Comment:
This is an easy fix. Will need to happen for stable.  This is may be a reason why unit tests are failing.